Tests/private_build/run_content_tests_private.py

```diff
@@ -22,7 +22,7 @@
     print_test_summary, update_test_msg, turn_off_telemetry, \
     create_result_files, get_all_tests, get_instances_ips_and_names, get_server_numeric_version, \
     initialize_queue_and_executed_tests_set, get_test_records_of_given_test_names, \
-    extract_filtered_tests, load_conf_files, set_integration_params, collect_integrations, notify_failed_test,\
+    extract_filtered_tests, load_conf_files, set_integration_params, collect_integrations, notify_failed_test, \
```

Comment:
how did we test the changes in this file? are @amshamah419 and @guykeller aware?

Reply:
The change here is a space added automatically